Files/Views/LayoutModes/GenericFileBrowser.xaml

```diff
@@ -614,6 +615,7 @@
             IsRightTapEnabled="True"
             ItemsSource="{x:Bind AssociatedViewModel.FilesAndFolders, Mode=OneWay}"
             LoadingRow="AllView_LoadingRow"
+            PointerPressed="{x:Bind local:App.CurrentInstance.InteractionOperations.List_ItemPress}"
```

Comment:
I think PointerPressed on DataGrid will register the event for the entire control rather than just the rows themselves. We should try to catch situations where the user may invoke something other than a Row this way.

Reply:
I cannot reproduce any issues around this.